crates/sovereign-sdk/adapters/mock-da/src/service.rs

```diff
@@ -190,22 +192,22 @@ impl MockDaService {
     pub async fn publish_test_block(&self) -> anyhow::Result<()> {
         let blocks = self.blocks.lock().await;
         let blob = vec![];
-        let _ = self.add_blob(&blocks, &blob, Default::default())?;
+        let _ = self.add_blob(&blocks, blob, Default::default())?;
         Ok(())
     }
 
     fn add_blob(
         &self,
         blocks: &AsyncMutexGuard<'_, DbConnector>,
-        blob: &[u8],
+        blob: Vec<u8>,
```

Comment:
I don't think this was necessary

Reply:
It wasn't. but its a mock da, so why not

Reply:
top kek